pxl_scripts/px/perf_flamegraph/trace.pxl

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+'''PEM Flamegraph
+
+Performance profile visualization.
+
+'''
+
+import px
+
+
+def stacktraces(start_time: str, node: str, namespace: str, pod: str, pct_grouping_entity: str):
+    df = px.DataFrame(table='stack_traces.beta', start_time=start_time)
+
+    df.namespace = df.ctx['namespace']
+    df.pod = df.ctx['pod']
+    df.container = df.ctx['container']
+    df.cmdline = df.ctx['cmdline']
+    df.node = df.ctx['node']
+
+    # This must be done before any filtering, to get accurate stack trace totals per node.
+    node_agg = df.groupby([pct_grouping_entity]).agg(
+        count=('count', px.sum)
+    )
+
+    # Apply filters.
+    df = df[px.contains(df.node, node)]
+    df = df[px.contains(df.pod, pod)]
+    df = df[px.contains(df.namespace, namespace)]
+
+    # Aggregate across time windows.
```

Comment:
IIUC, there aren't really time windows, just samples with timestamps. Maybe "Aggregate across all samples?"

Reply:
These are actually aggregating different flamegraphs into each other. For example, we dump a flamegraph every 30 seconds for the last 30-second time window. But if we query for the last five minutes, it'll take the ten flamegraphs and merge them into one larger flamegraph that covers the entire ten minute time window.

I'll update the comment to make that a bit clearer.